### ISSUE TITLE
07+12+COCO mAP is too low

### ISSUE THREAD

**Issue by zjuyinyue** [weiliu89/caffe#706]
hi,
I just cloned the ssd project and followed all the instructions in README, when I evaluated the models using score_ssd_pascal.py,  the first model SSD_300x300 gives reasonable result 77.6861% as follow,

I0811 23:11:34.654458 23677 caffe.cpp:155] Finetuning from models/VGGNet/VOC0712/SSD_300x300/VGG_VOC0712_SSD_300x300_iter_120000.caffemodel
I0811 23:11:34.767946 23677 net.cpp:761] Ignoring source layer mbox_loss
I0811 23:11:34.769688 23677 caffe.cpp:251] Starting Optimization
I0811 23:11:34.769696 23677 solver.cpp:294] Solving VGG_VOC0712_SSD_300x300_train
I0811 23:11:34.769713 23677 solver.cpp:295] Learning Rate Policy: multistep
I0811 23:11:34.818122 23677 solver.cpp:332] Iteration 0, loss = 2.42667
I0811 23:11:34.818158 23677 solver.cpp:433] Iteration 0, Testing net (#0) 
I0811 23:11:34.829445 23677 net.cpp:693] Ignoring source layer mbox_loss
I0811 23:12:55.971202 23677 solver.cpp:546]     Test net output #0: detection_eval = 0.776861
I0811 23:12:55.971362 23677 solver.cpp:337] Optimization Done.
I0811 23:12:55.971369 23677 caffe.cpp:254] Optimization Done.

while the corresponding COCO one(SSD_300x300_ft) gives only 75.6346% mAP, which does not make sense,

I0811 23:25:45.756788 24057 caffe.cpp:155] Finetuning from models/VGGNet/VOC0712/SSD_300x300_ft/VGG_VOC0712_SSD_300x300_ft_iter_120000.caffemodel
I0811 23:25:45.956925 24057 net.cpp:761] Ignoring source layer mbox_loss
I0811 23:25:45.960547 24057 caffe.cpp:251] Starting Optimization
I0811 23:25:45.960561 24057 solver.cpp:294] Solving VGG_VOC0712_SSD_300x300_ft_train
I0811 23:25:45.960564 24057 solver.cpp:295] Learning Rate Policy: multistep
I0811 23:25:46.010056 24057 solver.cpp:332] Iteration 0, loss = 2.58062
I0811 23:25:46.010082 24057 solver.cpp:433] Iteration 0, Testing net (#0)
I0811 23:25:46.021839 24057 net.cpp:693] Ignoring source layer mbox_loss
I0811 23:27:17.385046 24057 solver.cpp:546]     Test net output #0: detection_eval = 0.756346
I0811 23:27:17.385232 24057 solver.cpp:337] Optimization Done.
I0811 23:27:17.385238 24057 caffe.cpp:254] Optimization Done.

for the second result, I just changed " job_name = "SSD_{}".format(resize) "  in score_ssd_pascal.py to " job_name = "SSD_{}_ft".format(resize) ".

Could you give me any suggestions?

**Comment by weiliu89** [comment]
You should use the score_*.py script inside the coco model folder.

**Comment by RoseLii** [comment]
please help me .
I download VOCtrainval_06-Nov-2007，but when i running ./data/VOC0712/create_list.sh have error that
y@y-System-Product-Name:~/2T/wangxingli/caffe$ ./data/VOC0712/create_list.sh
Create list for VOC2007 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2012 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2007 test...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
I1025 21:36:51.506589 1277 get_image_size.cpp:61] A total of 0 images.
Why is this wrong?

**Comment by CrazyAlan** [comment]
@zjulujia Did you figure out the problem is? 
The 07+12+COCO SSD300* model was reported to have  81.2 on VOC2007 test, but the map of the released model is  0.756346 as @zjulujia said. Do you know what's wrong with it? @weiliu89 
I don't think it's the problem that you should run the evaluation in coco folder, since it's trying to evaluate on VOC2007 test

**Comment by CrazyAlan** [comment]
I used the score_ssd_pascal_ft.py file within the downloaded folder, then it worked
